### DISCUSSION TITLE
Issue with netbox-secretstore

### DISCUSSION THREAD

**Discussion by isaacmorse** [netbox-community/netbox#14225] [Help Wanted!]
I am running Netbox v3.4.10 and am trying to fix an issue with netbox-secretstore to help facilitate a migration to netbox-secrets. I've tried getting help for this by posting in the repos of the respective plugins, but have been unable to get a response.

The issue: I am able to view existing secrets, but unable to add new ones. When trying to add a new secret, I get this error:

```
<class 'netbox_secretstore.models.secrets.Secret.assigned_object_type.RelatedObjectDoesNotExist'>

Secret has no assigned_object_type.

Python version: 3.10.12
NetBox version: 3.4.10
```

The same issue looks to be preventing migrating to netbox-secrets, as it fails to import any secrets.

Any assistance with regards to resolving this would be greatly appreciated.

**Comment by deku-m** [comment]
Did you do the following on the end?
These instructions assume that you are running Netbox v3.4.x and the plugin version 1.7.x. Install a new version of netbox_secretstore as:
- pip install git+https://github.com/Onemind-Services-LLC/netbox-secretstore@migration/nb34

Note: You should have netbox-secretstore v1.4.4 installed now.
Make sure to add both plugins to the configuration.py before the migration.
Run the migration:

- python manage.py migrate

Finally, readjust the indices for the netbox-secrets plugin
- python manage.py sqlsequencereset netbox_secrets

Run the output of the previous command in the database.
You can now remove netbox-secretstore from the application. You may have clean up your database of the old tables manually.
